packages/pwa-kit-react-sdk/src/ssr/universal/utils.ts

```diff
@@ -56,31 +53,24 @@ export const getAssetUrl = (path: string) => {
  * @function
  * @returns {string} The full URL to the static asset.
  */
-export const getStaticAssetUrl = (path = '', opts: GetAssetUrlOptions) => {
+export const getStaticAssetUrl = (path: string, opts: GetAssetUrlOptions) => {
```

Comment:
nit: Do we want to keep providing a default values to handle undefined inputs?

```suggestion
export const getStaticAssetUrl = (path: string = '', opts: GetAssetUrlOptions = {}) => {
```